main.go

```diff
@@ -10,25 +11,21 @@ import (
 // get the current listening address or fail if input is not correct
 func determineListenPort() (string, error) {
     port := os.Getenv("PORT")
-  
+
     if port == "" {
         return "", fmt.Errorf("$PORT not set")
     }
-  
+
     return ":" + port, nil
 }
 
-func rootHandler(
+func handleGetRootRequest(
         responseWriter http.ResponseWriter,
         request *http.Request) {
 
-    var responseContent string = fmt.Sprintf(
-        "Hi there, I love %s!",
-        request.URL.Path[1:])
-
-    fmt.Fprintf(
-        responseWriter,
-        responseContent)
+    // TODO handle error
+    parsedTemplate, _ := template.ParseFiles("root.tmpl")
```

Comment:
I see you have the todo, but can we default to just log errors? I just don't like silent bugs, i'd rather have to sift through too many logs than have to find an error that is being swallowed

Reply:
will do. though how do we decide when to `log.Fatal()` vs `panic()`?

Reply:
I think panic should be avoided unless the program literally cannot continue. 

https://www.reddit.com/r/golang/comments/2a1sie/when_to_call_panic/ciqn81u/

Reply:
okay we've gotta figure out a succinct way of returning a 500 in these situations